Antidote/FriendsViewController.m

```diff
@@ -80,17 +80,6 @@ - (void)viewWillAppear:(BOOL)animated
     [self updateBarButtonItem];
 
     [self updateSegmentedControlRequestTitle];
-
-    UIImageView *hairlineImageView = [self findHairlineImageViewUnder:self.navigationController.navigationBar];
-    hairlineImageView.hidden = YES;
-}
-
-- (void)viewWillDisappear:(BOOL)animated
-{
-    [super viewWillDisappear:animated];
-
-    UIImageView *hairlineImageView = [self findHairlineImageViewUnder:self.navigationController.navigationBar];
```

Comment:
You have removed all usages of `findHairlineImageViewUnder` so there is no need in it now. You can delete method itself.